sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/exemplar/FixedSizeExemplarReservoirTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.exemplar;
+
+import static io.opentelemetry.sdk.testing.assertj.metrics.MetricAssertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.api.trace.TraceFlags;
+import io.opentelemetry.api.trace.TraceState;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.common.RandomHolder;
+import io.opentelemetry.sdk.testing.time.TestClock;
+import java.time.Duration;
+import java.util.Random;
+import org.junit.jupiter.api.Test;
+
+public class FixedSizeExemplarReservoirTest {
```

Comment:
Jupiter, so no public class / method per jupiter idiom

Reply:
still not used to jupiter, fixing.